utils/ssh.py

```diff
@@ -103,9 +131,9 @@ def run_command(self, command):
                     break
             exit_status = session.recv_exit_status()
             return SSHResult(exit_status, output)
-
-        # Returning two things so tuple unpacking the return works even if the ssh client fails
-        return SSHResult(None, None)
+        except paramiko.SSHException:
+            # Returning two things so tuple unpacking the return works even if the ssh client fails
+            return SSHResult(1, None)
```

Comment:
Woops. This is probably a bug. Probably should catch and log, and leave the sentinel return the way it is.